.prettierrc

```diff
@@ -2,5 +2,6 @@
   "tabWidth": 2,
   "useTabs": false,
   "semi": true,
-  "bracketSpacing": true
-}
+  "bracketSpacing": true,
+  "printWidth": 120
```

Comment:
oh this was an intentional change I was making. I had assumed we were already following a line limit of `120` and thought it would be a good idea to make that an explicit configuration. Am I wrong in that assumption? 

Either way, if we're following `80` or `100` or `120, I think it's a good idea to explicitly state that in the .prettierrc to keep our code aligned right? I'm personally more comfortable with `120` since most devs nowadays are using wide screens.

I think it's also recommended to use more than 80, but open to using that. For reference: https://prettier.io/docs/en/options.html#print-width

Reply:
Coming back to this, looked at the rest of the codebase and it does look like we're using `80` for everything. I aligned my changes to that, I think it makes sense to keep precedent.

I also explicitly listed `80` in the `.prettierrc` file, I think I had mine set to `120` which was confusing me.